dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -97,79 +168,448 @@ export const DynamicPackage = ({
 									containerType="dynamic/package"
 									showAge={showAge}
 									supportingContent={card.supportingContent}
+									imageUrl={undefined}
 								/>
 							</LI>
 						);
 					})}
 				</UL>
-			</>
-		);
-	}
+			</LI>
+		</UL>
+	);
+};
+
+const Card25_Card25_ColumnOfTwo25_ColumnOfTwo25 = ({
+	cards,
+	containerPalette,
+	showAge,
+}: {
+	cards: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+}) => {
+	const bigs = cards.slice(0, 2);
+	const columns = [cards.slice(2, 4), cards.slice(4, 6)];
+
 	return (
-		<>
-			{!!snap && (
-				<Snap100
-					snap={snap}
+		<UL direction="row">
+			{bigs.map((card) => {
+				return (
+					<LI
+						key={card.url}
+						padSides={true}
+						showTopMarginWhenStacked={false}
+						padBottom={false}
+						padBottomOnMobile={true}
+						percentage="25%"
+						showDivider={true}
+					>
+						<FrontCard
+							trail={card}
+							containerPalette={containerPalette}
+							containerType="dynamic/package"
+							showAge={showAge}
+							supportingContent={card.supportingContent}
+						/>
+					</LI>
+				);
+			})}
+			{columns.map((columnCards) => {
```

Comment:
👍 this reads nice

Reply:
I just changed this because it didn't work but thankyou 😢